### PR TITLE
Add new regions and carriers (bug 1151991, bug 1153002, bug 1154951, bug  1157067)

### PR DESCRIPTION
**Pull request by diox**
https://bugzilla.mozilla.org/show_bug.cgi?id=1151991
https://bugzilla.mozilla.org/show_bug.cgi?id=1153002
https://bugzilla.mozilla.org/show_bug.cgi?id=1154951
https://bugzilla.mozilla.org/show_bug.cgi?id=1157067

### CODE REVIEW

File: src/media/js/mobilenetwork.js [hunk 1]
```diff
@@ -129,6 +105,11 @@ define('mobilenetwork',
             20: 'america_movil'
         },
 
+        // Jordan
+        416: {
+            77: 'orange'
```
Comment (ngokevin) [inline src/media/js/mobilenetwork.js]: Might save some bytes if we store the strings as constants

File: src/media/js/mobilenetwork.js [hunk 2]
```diff
@@ -1,34 +1,10 @@
 define('mobilenetwork',
-    ['core/l10n', 'core/log', 'regions', 'tracking', 'core/user', 'core/utils'],
-    function(l10n, log, regions, tracking, user, utils) {
+    ['core/l10n', 'core/log', 'carriers', 'regions', 'tracking', 'core/user', 'core/utils'],
```
Comment (ngokevin) [inline src/media/js/mobilenetwork.js]: abc

**Comment by ngokevin** [comment]
r+wc, if one of my unit tests fail, i'll fix it. seems to be intermittent

File: src/media/js/mobilenetwork.js (hunk 1 above)
Comment (diox) [inline src/media/js/mobilenetwork.js]: I think it's negligible post-gzip, yeah. Ultimately the proper fix would be to move the mcc/mnc to the carriers module I created in marketplace-constants but that was a lot of work that didn't bring much improvement so I decided against it for this PR.

**Pull request merged**